podkrepi-bg/frontend
# feat: Add archived filter to the campagin application list

Closes #1981

## Motivation and context

- Added toggle button to show/hide archived issues
- Used the same design as in the active campaigns grid ( `/admin/campaigns` ) - toolbar and button style:
![image](https://github.com/user-attachments/assets/f5610f34-bfee-4e5a-afe8-77578e5ed94e)







## Screenshots:



Before|After
---|---
![image](https://github.com/user-attachments/assets/3e6ff194-97b6-4f05-82f7-aceeee54c1ee)|![image](https://github.com/user-attachments/assets/7ed1e3f2-1271-4876-99d5-69d6bf3b52fc)



## Testing

### Steps to test
Open `admin/campaign-applications` 
Try the "Show / Hide archived" button
Grid list should be filtered out accordingly
### Affected urls
`admin/campaign-applications` 







## Environment

### New environment variables:




### New or updated dependencies:



Dependency name|Previous version|Updated version|Details
---|---|---|---
`dependency/name`|`v1.0.0`|`v2.0.0`|
